docker/Dockerfile

```
@@ -93,46 +90,67 @@ RUN python /usr/local/generate_toolchain.py --prefix /usr/local
 RUN install-from-source https://www.zlib.net/zlib-1.2.11.tar.gz
 
 # LLVM and Clang
-RUN BUILD_FILE_DIR='llvm' install-cmake-from-source \
-    https://github.com/llvm/llvm-project/archive/llvmorg-9.0.1.tar.gz \
-    -DLLVM_ENABLE_PROJECTS="clang"
+RUN BUILD_FILE_DIR='llvm' install-cmake-from-source https://github.com/llvm/llvm-project/archive/llvmorg-11.0.1.tar.gz \
+    -DLLVM_ENABLE_PROJECTS="clang" \
```

You should put clang-tools-extra here so you get clang-tidy
```suggestion
    -DLLVM_ENABLE_PROJECTS="clang;clang-tools-extra" \
```

---

I was planning to add that in a future PR where I pull in the proper `.clang-tidy` file and make the changes necessary for it to run

---

Fair enough